media-sound/flac123/flac123-0.0.12.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A command-line program for playing FLAC audio files"
+DESCRIPTION="flac-tools aims to be a complete set of tools for playing and \
+manipulating audio files stored in the FLAC format.  flac123 is the only \
```

Comment:
You can squeeze (into 1 white space) the 2 white spaces between "format." and "flac123".